src/main/java/core/basesyntax/service/impl/ReaderFromCsvImpl.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.ReaderFromCsv;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+
+public class ReaderFromCsvImpl implements ReaderFromCsv {
+
+    @Override
```

Comment:
The method implementation should not begin with an empty line. Please remove the empty line to comply with the checklist.